controllers/bsl.go

```diff
@@ -285,10 +286,14 @@ func (r *DPAReconciler) validateAWSBackupStorageLocation(bslSpec velerov1.Backup
 		return fmt.Errorf("prefix for AWS backupstoragelocation object storage cannot be empty. It is required for backing up images")
 	}
 
-	// BSL region is required when s3ForcePathStyle is true OR BackupImages is true
-	// TODO: Update logic when (region auto-discovery) https://github.com/openshift/openshift-velero-plugin/pull/223 is merged
-	if (bslSpec.Config == nil || len(bslSpec.Config[Region]) == 0) && (bslSpec.Config != nil && bslSpec.Config[S3ForcePathStyle] == "true" || dpa.BackupImages()) {
-		return fmt.Errorf("region for AWS backupstoragelocation cannot be empty when s3ForcePathStyle is true or when backing up images")
+	// BSL region is required when
+	// - s3ForcePathStyle is true, because some velero processes requires region to be set and is not auto-discoverable when s3ForcePathStyle is true
+	//   imagestream backup in openshift-velero-plugin now uses the same method to discover region as the rest of the velero codebase
+	//   after https://github.com/openshift/openshift-velero-plugin/pull/223.
```

Comment:
NIT: Lets remove this comment as the PR is merged ?

Reply:
ack